### DISCUSSION TITLE
Samba and nfs

### DISCUSSION THREAD

**Discussion by robertocannito** [SpiralLinux/SpiralLinux-project#20] [Q&A]
I would like install spiral Linux gnome edition could you give me information how how configure samba and Nas?

**Comment by geckolinux** [comment]
Hi there, the process would be the same as for vanilla Debian:
https://wiki.debian.org/Samba/

And to create a NAS I would recommend [OpenMediaVault](https://www.openmediavault.org), which also uses Debian as its base.
